fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -579,17 +579,7 @@ public boolean handleHbResponse(BackendHbResponse hbResponse, boolean isReplay)
             }
 
             this.lastUpdateMs = hbResponse.getHbTime();
-            if (!isAlive.get()) {
```

Comment:
Can you give an example of a scenario that triggers the bug?

Reply:
BE node is NOT dead but under heavy load that it can't respond the FE's heartbeat rpc call. FE receives a few consecutive fail heartbeat fails and set the BE not alive. After the BE restore to its normal state, it can respond the RPC again correctly. Intermittent network issue between FE and the target BE can lead to the same result.

From BE point of view, it is just not responding for a while, not a reboot.